src/core/stream.c

```diff
@@ -575,6 +567,7 @@ QuicStreamParamSet(
     case QUIC_PARAM_STREAM_ID:
     case QUIC_PARAM_STREAM_0RTT_LENGTH:
     case QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE:
+    case QUIC_PARAM_STREAM_BLOCKED_TIMINGS:
         Status = QUIC_STATUS_INVALID_PARAMETER;
         break;
```

Comment:
No need to make this change, but there's no point in having all these when the `default` already does this...

Reply:
Weird. All these parameters should not have been added here then.